src/app/copilot/components/copilot-panel/copilot-panel.component.scss

```
@@ -0,0 +1,1561 @@
+      }
+    }
+
+    &__label {
+      font-weight: 500;
+      flex-shrink: 0;
+    }
+
+    &__value {
+      font-weight: 600;
+      text-align: right;
+      word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)